effekt/shared/src/main/scala/effekt/core/Parser.scala

```diff
@@ -105,7 +105,10 @@ class CoreParsers(positions: Positions, names: Names) extends EffektLexers(posit
   // Definitions
   // -----------
   lazy val definition: P[Definition] =
-  ( `let` ~> id ~ (`=` ~/> expr) ^^ Definition.Let.apply
+  ( `let` ~> id ~ opt(`:` ~/> valueType) ~ (`=` ~/> expr) ^^ {
```

Comment:
there should already by an `optionalType` or similar.

Reply:
See line 126 case `Val`

Reply:
Thanks, will do.